src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/Enforcement.java

```diff
@@ -36,6 +37,7 @@
 @Entity
 @Table(name = "ENFORCEMENT")
 @Where(clause = "SOFT_DELETED = 0")
+@EqualsAndHashCode(callSuper=false)
```

Comment:
Is this change related to the work?

Reply:
we need to look into , definitely need to be changed